src/client/opamAction.ml

```diff
@@ -84,8 +82,8 @@ let preprocess_dot_install_t st nv build_dir =
       in
       [dir, inst]
     in
-    dir_and_install @
-    List.map (fun (base, dst) ->
+    dir_and_install @ List.rev @@
+    List.rev_map (fun (base, dst) ->
```

Comment:
```suggestion
    dir_and_install @
    List.map (fun (base, dst) ->
```
I went into *let's tail rec everything* mode, but maybe this is too much optimisation? It handles the case where there is a huge amount of files in a given field field in the `.install` file

Reply:
I think it's a good idea, actually - thank you for doing it!